tests/acceptance/acceptancesuite.js

```diff
@@ -63,6 +63,31 @@ test('navigating and refreshing mantains that page number', async t => {
   await t.expect(pageNum).eql('Page 2');
 });
 
+const spellCheckLogger = RequestLogger({
+  url: /v2\/accounts\/me\/answers\/vertical\/query/
+});
+test.requestHooks(spellCheckLogger)('spell check flow', async t => {
+  const searchComponent = VerticalPage.getSearchComponent();
+  await searchComponent.enterQuery('varginia');
+  await searchComponent.submitQuery();
+
+  // Check that clicking spell check resets pagination to page 1
+  const paginationComponent = VerticalPage.getPaginationComponent();
+  await paginationComponent.clickNextButton();
+  let pageNum = await paginationComponent.getActivePageLabelAndNumber();
+  await t.expect(pageNum).eql('Page 2');
+
+  const spellCheckComponent = VerticalPage.getSpellCheckComponent();
+  await spellCheckComponent.clickLink();
+  pageNum = await paginationComponent.getActivePageLabelAndNumber();
+  await t.expect(pageNum).eql('Page 1');
+
+  // Check that clicking spell check sends a queryTrigger=suggest url param
```

Comment:
This part of the test feels like it should be an integration test rather than an acceptance test because it is not directly related to user flow. What are your thoughts?

Reply:
As with our other item this Sprint, if we can't easily express this as an integration test (or unit test), I think we should keep it in the acceptancesuite for now. Make a TODO though indicating the test should be migrated once we improve our unit/integration testing abilities. This might be hard to do as anything but an acceptance test. I think we'd need to be able to mock the Core dependency given to the SpellCheck component.

Reply:
I can add a todo! I tried to think of ways to test this in jest, and I think it would be kind of difficult, since clicking the spell check suggestion (like varginia -> virginia) does a full page refresh and reloads all of the components. Probably still something we can set up in jest but requires a little more thought I think.